sty/omtext/omtext.dtx

```diff
@@ -329,7 +329,9 @@ use LaTeXML::Util::Pathname;
 \ifindex\makeindex\fi
 %</package>
 %<*ltxml>
+DeclareOption('showmeta',sub {PassOptions('metakeys','sty',ToString(Digest(T_CS('\CurrentOption')))); });
 DeclareOption('noindex','');
+ProcessOptions();
```

Comment:
You should also add this to the .ltxml file. The cross-maintaining of .dtx and .ltxml always has me confused, personally, it's easy to overlook one or the other.